Makefile

```diff
@@ -67,7 +67,7 @@ api-1.30.0-template:
 
 .PHONY: api-1.30.0-kubeconform
 api-1.30.0-kubeconform: $(KUBECONFORM)
-	$(KUBECONFORM) -kubernetes-version 1.30.0 -summary -strict charts/ocis/ci/templated.yaml
+	$(KUBECONFORM) -kubernetes-version 1.30.0 -schema-location default -schema-location 'https://raw.githubusercontent.com/garethr/openshift-json-schema/master/v4.1.0-standalone{{ .StrictSuffix }}/{{ .ResourceKind }}.json' -summary -strict charts/ocis/ci/templated.yaml
```

Comment:
this openshift version is already 5 years old. This repo seems to unmaintained, please also have a look at https://github.com/owncloud/ocis-charts/pull/653/files#r1706861465

Reply:
we added this because i had a mistake 

`/go/bin/kubeconform-v0.6.6 -kubernetes-version 1.27.0 -summary -strict charts/ocis/ci/templated.yaml
charts/ocis/ci/templated.yaml - Route collaboration-collabora failed validation: could not find schema for Route
charts/ocis/ci/templated.yaml - Route collaboration-onlyoffice failed validation: could not find schema for Route
Summary: 169 resources found in 1 file - Valid: 167, Invalid: 0, Errors: 2, Skipped: 0
make: *** [Makefile:37: api-1.27.0-kubeconform] Error 1`

Could you advise something in this case?